spec/amp-3p-video.md

```diff
@@ -0,0 +1,82 @@
+# Vendor-specific (3p) video player components
+
+The generic [`amp-video`](https://go.amp.dev/c/amp-video) component plays videos directly on a page, much like a `<video>` tag.
+
+Some video cannot be embedded this way. AMP provides vendor-optimized components like [`amp-youtube`](https://go.amp.dev/c/amp-youtube),
+[`amp-ima-video`](https://go.amp.dev/c/amp-ima-video), amongst [others](./amp-video-interface.md).
+Under the hood, these load an iframe whose page communicates with the outer document to coordinate playback.
+
+For custom video implementations, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) is a generic component that can load any video document to coordinate its playback.
```

Comment:
Perhaps something like...

> We would prefer to not have additional custom video implementations. Instead we believe, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) should be used instead, since it is a generic component that can load any video document to coordinate playback.